e2e/lib/k8up.bash

```diff
@@ -129,10 +158,64 @@ given_a_running_operator() {
 	echo "✅  A running operator is ready"
 }
 
+given_an_existing_backup() {
+	if [ "${#}" != 2 ]; then
+		errcho "$0 Expected 2 arguments, got ${#}."
+		exit 1
+	fi
+
+	backup_file_name=${1}
+	backup_file_content=${2}
+	given_a_subject "${backup_file_name}" "${backup_file_content}"
+
+	apply definitions/backup
+	wait_until backup/k8up-k8up-backup completed
+
+	run restic dump latest "/data/subject-pvc/${backup_file_name}"
+	# shellcheck disable=SC2154
+	[ "${backup_file_content}" = "${output}" ]
+
+	echo "✅  An existing backup is ready"
+}
+
 wait_until() {
 	object=${1}
 	condition=${2}
 	ns=${NAMESPACE=${DETIK_CLIENT_NAMESPACE}}
 	echo "Waiting for '${object}' in namespace '${ns}' to become '${condition}' ..."
 	kubectl -n "${ns}" wait --timeout 1m --for "condition=${condition}" "${object}"
 }
+
+expect_file_in_container() {
+	if [ "${#}" != 4 ]; then
+		errcho "$0 Expected 4 arguments, got ${#}."
+		exit 1
+	fi
+
+	pod=${1}
+	container=${2}
+	expected_file=${3}
+	expected_content=${4}
+
+	commands=(
+		"ls -la \"$(dirname "${expected_file}")\""
+		"test -f \"${expected_file}\""
+		"cat \"${expected_file}\""
+		"test \"${expected_content}\" \"=\" \"\$(cat \"${expected_file}\")\" "
+	)
+
+	echo "Testing if file '${expected_file}' contains '${expected_content}'"
+	echo "in container '${container}' of pod '${pod}'."
```

Comment:
I would advise to keep single log statement in single echo call. Our target screens aren't mobile phones ;)
```suggestion
	echo "Testing if file '${expected_file}' contains '${expected_content}' in container '${container}' of pod '${pod}'."
```